rocky/reports/forms.py

```diff
@@ -47,25 +57,32 @@ class ReportScheduleStartDateForm(BaseRockyForm):
         input_formats=["%Y-%m-%d"],
     )
 
-
-class ReportRecurrenceChoiceForm(BaseRockyForm):
-    choose_recurrence = forms.ChoiceField(
-        label="",
-        required=False,
-        widget=forms.RadioSelect(attrs={"class": "submit-on-click"}),
-        choices=(("once", _("No, just once")), ("repeat", _("Yes, repeat"))),
-        initial="once",
+    start_time = forms.TimeField(
```

Comment:
there is also `models.DateTimeField` to have both Date and time together in the widget.

Reply:
<img width="721" alt="image" src="https://github.com/user-attachments/assets/dd4a3e32-0c9c-4e4e-be43-33ac2f40e1dd">

Reply:
Are we converting the local time to UTC? I think it is strange that when using now time time the user needs to think of 2 hours in the past ?